backend/src/server/routes/v3/secret-router.ts

```diff
@@ -300,6 +300,11 @@ export const registerSecretRouter = async (server: FastifyZodProvider) => {
         secretPath: z.string().trim().default("/").transform(removeTrailingSlash).describe(RAW_SECRETS.GET.secretPath),
         version: z.coerce.number().optional().describe(RAW_SECRETS.GET.version),
         type: z.nativeEnum(SecretType).default(SecretType.Shared).describe(RAW_SECRETS.GET.type),
+        expandSecretReferences: z
+          .enum(["true", "false"])
+          .default("false")
```

Comment:
what do you think of defaulting to `true` instead? 

I think from a client POV, they ideally shouldn't have to worry about what secret references are and whether they should be expanded

Reply:
@sheensantoscapadngan Yea I personally agree that we should default it to true. However to keep it inline with our existing implementation for the get all secrets endpoint, I opted to make it default to false. I feel like if we're going to make it default to true, it should also default to true on the list secrets endpoint

Reply:
yea... It becomes inconsistent if we start doing it this way which could be confusing. let's keep this as it is